rust/definitions/src/navigation.rs

```diff
@@ -430,6 +430,35 @@ pub enum MSCContent {
     AddSpecs { f: MSCNetworkInfo },
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum QrData {
```

Comment:
I"m not sure that backend should decide which qr code is sensitive it it's model. 
I will ignore this information on android and will always make qr code in private key screen look sensitive and in key details - normal.

Reply:
I think it's fine, as this is like property in database model telling you property of a given object, it does not tell you how to display it.